src/i18n.ts

```diff
@@ -28,8 +24,10 @@ export interface I18NEventPayload {
 
 export const I18N_EA_SIGNAL = "i18n:locale:changed";
 
-@autoinject()
 export class I18N {
+
+  public static inject() { return [EventAggregator, BindingSignaler]; }
```

Comment:
We had issues with `@inject` or `static inject = [..]` and `DOM.Element` combination, because the code was evaluated earlier than platform specific code for initialization, often `initialize()`. Also I think `@autoinject` requires metadata to work with, while it works fine this way.

Reply:
Thanks for refreshing my mind :)